### ISSUE TITLE
Duplicate Content Warnings

### ISSUE THREAD

**Issue by seandawes** [solidusio-contrib/solidus_reviews#50]
The native functionality of this extension creates a bad experience for SEO and crawlers. Currently any product page with review functionality is creating a new url of /products/PRODUCT/reviews/new

Due to this new page having no content it creates un-needed pages which flag for dup content. Ideally this should not exist and when clicking to add a review it should just be handled on the product page

Example:
![image](https://user-images.githubusercontent.com/1217031/63617605-e1d48880-c5b7-11e9-9bfc-1c0483484f75.png)

**Comment by seandawes** [comment]
keep active

**Comment by fthobe** [comment]
I agree, it's not the way it should work. The correct way would be to introduce a canonical to the main page or remove the page what so ever. 

@tvdeyen , Is there any particular reason why the reviews create pages? It seems like an optional feature.
